### PR TITLE
wrong video deleted (hopefully)

### PR DESCRIPTION
**Pull request by nomadscientist**

### CODE REVIEW

File: _data/materials.yaml [hunk 1]
```diff
@@ -1534,12 +1534,6 @@ transcriptomics-scrna-intro-slides:
   type: Lecture
   tags: [transcriptomics, single-cell]
   videos:
-  - captions:
```
Comment (hexylena) [inline _data/materials.yaml:1537]: no, the other one! you want to get rid of 'smorgasbord' not spanscriptomics

Comment (nomadscientist) [reply _data/materials.yaml:1537]: whoops on it

Comment (nomadscientist) [reply _data/materials.yaml]: hmmm

Comment (nomadscientist) [reply _data/materials.yaml]: are you certain?

Comment (nomadscientist) [reply _data/materials.yaml]: https://youtu.be/D3qvYWZMFa0 this one works whereas the hR4SpWSNUSY doesn't...

**Review by hexylena — APPROVED** [review]
Yes this looks good now

File: _data/materials.yaml (hunk 1 above)
Comment (hexylena) [reply _data/materials.yaml]: Ohhh I see it's private. fixed.

Comment (hexylena) [reply _data/materials.yaml]: (if you can confirm.)

**Pull request merged**